public/app/features/dashboard-scene/utils/getNextPanelId.ts

```diff
@@ -0,0 +1,59 @@
+import { SceneGridItem, SceneGridLayout, SceneGridRow, SceneObject, VizPanel } from '@grafana/scenes';
+
+import { DashboardScene } from '../scene/DashboardScene';
+import { PanelRepeaterGridItem } from '../scene/PanelRepeaterGridItem';
+
+import { getPanelIdForVizPanel } from './utils';
+
+export class NextPanelIdGenerator {
```

Comment:
@torkelo  - Not related to this PR I think?

Reply:
@dprokop oops, no that was a test from yesterday :)